src/cmd.c

```diff
@@ -2147,6 +2157,7 @@ static const atci_command_t cmds[] = {
     {"$CW",          cw,      NULL,             NULL,             NULL, "Start continuous carrier wave transmission"},
     {"$CM",          cm,      NULL,             NULL,             NULL, "Start continuous modulated FSK transmission"},
     {"$NVM",         NULL,    set_nvm,          NULL,             NULL, "Store / Read data from Non Volatile Memory"},
+    {"$APKACCESS",   protect_appkey, NULL,      NULL,             NULL, "Protect AppKey against read access"},
```

Comment:
I agree, this should also applied to AppSkey and NetSkey

Reply:
If that's not secret, what's your use case for AT$APKACCESS? Did you have any specific attacks or threats in mind?

Reply:
The idea is to protect the keys and make access to the most private  one "AppKey" as  much  complex as possible. As the modem is saving it into the eeprom you don't need to store it in the MCU it is really great, it can be removed from the MCU firmware with a dynamic loading, perfect.
But it's  really easy to send an AT command direectly on the modem line from an external system to extract it. So if the AT command can be blocked, the only way is to attack the eeprom, this is more complex. To go to the  next step, you can take a look at my software secure store for keys in my abz sdk on my github.
Here I don't wont to implement a such security level but it's  just to avoid a so  simple way  to extract keys.

Honestly, i'm not really think about a real protection way with this command, but it's to avoid all the "security researchers" and other script kiddies mocking on IoT because they discovered how to send a bit on a serial line wire... so it's "best practice" to make a such minimal protection on credentials.